src/whylogs/app/logger.py

```diff
@@ -168,15 +212,31 @@ def flush(self,rotation_suffix:str =None):
             return None
 
         for writer in self.writers:
-            if rotation_suffix is None:
-                writer.write(self._profiles[-1])
-            else:
-                writer.write(self._profiles[-1],rotation_suffix)
-        
+            # write full profile
 
-    # def load_from_file():
+            if self.full_profile_check():
+                
+                if rotation_suffix is None:
+                    writer.write(self._profiles[-1]["full_profile"])
+                else:
+                    writer.write(self._profiles[-1]["full_profile"],rotation_suffix)
+
+            if self.segments is not None:
+
+                for hashseg, each_seg_prof in self._profiles[-1]["segmented_profiles"].items():
+                    seg_suffix = hashseg
+                    full_suffix= "_"+seg_suffix
+                    if rotation_suffix is None:
+                        writer.write(each_seg_prof,full_suffix)
```

Comment:
spacing? our linter doesn't check for this?

Reply:
I wil check, I have been finding these everywhere too.